rootston/cursor.c

```diff
@@ -98,51 +99,56 @@ static void seat_view_deco_button(struct roots_seat_view *view, double sx,
 	}
 }
 
-static void roots_cursor_update_position(struct roots_cursor *cursor,
+static void roots_passthrough_cursor(struct roots_cursor *cursor,
 		uint32_t time) {
-	struct roots_desktop *desktop = cursor->seat->input->server->desktop;
-	struct roots_seat *seat = cursor->seat;
 	struct roots_view *view;
```

Comment:
```c
struct roots_view *view = NULL;
```

Reply:
nack, it's set by desktop_surface_at

Reply:
It's not obvious that it's set to NULL if there's no view.